twint/user.py

```diff
@@ -18,17 +18,21 @@ def __init__(self):
 # ur object must be a json from the endpoint https://api.twitter.com/graphql
 def User(ur):
     logme.debug(__name__ + ':User')
-    if 'data' not in ur and 'user' not in ur['data']:
+    if 'data' not in ur or 'user' not in ur['data']:
         msg = 'malformed json! cannot be parsed to get user data'
+        if 'errors' in ur:
+            msg = ur['errors'][0]['message']
         logme.fatal(msg)
-        raise KeyError(msg)
+        raise RuntimeError(msg)
     _usr = user()
```

Comment:
Should all of the below `_usr` fields have a default that's not None?

Reply:
Sorry, I am unable to understand what you mean here,
Could you elaborate a bit?

Reply:
Apologies. Please ignore this comment. I don't think it's necessary. Thanks